addons/website_sale_delivery/views/website_sale_delivery_templates.xml

```diff
@@ -1,6 +1,21 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
-
+    <template id="publish_management">
```

Comment:
Find a better way to add a class, or do not add it ... I bet you widget will be broken in a few months when someone updates the subscription js and nobody will notice it.